### PR TITLE
Add support for UISegmentedControl.

### PR DESCRIPTION
**Pull request by vanaudiobuddy**
This includes:
- Refactoring UI elements to separate UIViewController subclasses. There is no need for the Style singleton to hold on to those UI elements.
- Adding support for UISegmentedControl for the no-python StyleKit.
- Removing some font names in the JSON file that do not exist (MarkerFelt-Bold, MarkerFelt-Black, MarkerFelt-LightItalic).

### CODE REVIEW

File: SwiftStyleKitSample/StyleKitSample/ButtonsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,47 @@
+//
```
Comment (ekille) [inline SwiftStyleKitSample/StyleKitSample/ButtonsViewController.swift]: We generally remove all header comments

File: SwiftStyleKitSample/StyleKitSample/ButtonsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+//  ButtonsViewController.swift
+//  StyleKitSample
+//
+//  Created by Van Nguyen on 4/12/16.
+//  Copyright © 2016 Tonic Design. All rights reserved.
+//
+
```
Comment (ekille) [inline SwiftStyleKitSample/StyleKitSample/ButtonsViewController.swift]: We generally remove all header template text

File: SwiftStyleKitSample/StyleKitSample/ButtonsViewController.swift [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    /*
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        // Get the new view controller using segue.destinationViewController.
+        // Pass the selected object to the new view controller.
+    }
+    */
```
Comment (ekille) [inline SwiftStyleKitSample/StyleKitSample/ButtonsViewController.swift]: General Tonic thing, leave no code commented out

File: SwiftStyleKitSample/StyleKitSample/Constants.swift [hunk 4]
```diff
@@ -8,6 +8,11 @@
 
 import Foundation
 
+struct FontProperties {
+    static let Name = "font"
+    static let Size = "size"
+}
```
Comment (ekille) [inline SwiftStyleKitSample/StyleKitSample/Constants.swift:75]: Should we use and enum here?

Comment (vanaudiobuddy) [inline SwiftStyleKitSample/StyleKitSample/Constants.swift:75]: Definitely!

Comment (vanaudiobuddy) [inline SwiftStyleKitSample/StyleKitSample/Constants.swift:75]: Actually, I'm not so sure. If I use enum, I will end up writing rawValue everywhere, which can be tedious. That's why I used structs with constant strings from the first place.

**Pull request merged**